### ISSUE TITLE
Improve the "Documentation" section of the README

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5-vue#45]
Like in https://github.com/ckeditor/ckeditor5-angular:

![image](https://user-images.githubusercontent.com/156149/56117102-ae2d8600-5f67-11e9-9337-f7187f206463.png)

**Comment by ragaar** [comment]
_While I would value a response, do not feel as though it is mandatory until you have had time to work on this issue. I can read instructions and use Google._ Putting an answer here would be more for everyone else on the internet that stumbles into this later.
&nbsp;

I just stumbled into this about an hour ago and there is a ton of documentation, but it feels like it is a struggle to follow the links to the place to get everything installed.

Getting CKEditor installed wasn't bad, but how do want someone to link Vue to it? Giving a step-by-step kind of approach would be a handy reference tool to get more adoption.

&nbsp;
Presently, my path has been:
> [Quick start](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/frameworks/vuejs.html#quick-start)<sup>[1]</sup>
>> Install the [CKEditor 5 WYSIWYG editor component for Vue.js](https://www.npmjs.com/package/@ckeditor/ckeditor5-vue)<sup>[2]</sup> and the editor build of your choice.

At which point, I am looking at this and saying "what".

Hopefully this helps provide some insight into someone with no exposure to this product!
&nbsp;

REFERENCES:
1. https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/frameworks/vuejs.html#quick-start
2. https://www.npmjs.com/package/@ckeditor/ckeditor5-vue

**Comment by oleq** [comment]
@ragaar Have you actually read https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/frameworks/vuejs.html#quick-start? There are plenty examples in there. 

> Quick start[1]
> Okay, I'll install the ZIP file.
> index.html loads nicely and I can play around with stuff inside of that.
> Install the CKEditor 5 WYSIWYG editor component for Vue.js[2] and the editor build of your choice.

TBH, I'm a little bit confused by your problem description. Could you tell us more maybe?

**Comment by ragaar** [comment]
The link you cited is the URL tied to Quick Start and referenced in the citations. That was exactly where I was starting from.

So, let's look at this page.
The first statement highlights that a Vue.js component for CKEditor needs to be installed. Click on the provided link and I am sent to an NPM page for @ckeditor/ckeditor5-vue. The documentation on here takes you back to the page we just came from. **Concern:** This creates a circular reference if you're trying to follow the document in order.

Moving forward, the next step is to install the editor build of your choice. I see this as a different project, so I won't go into the details here. But, as I highlighted previously, I installed the balloon editor from the ZIP file provided as a release.

Next, I'm told -- using npm -- to install build-classic and the ckeditr5-vue component. **Concern:** This is an arbitrary path that not every user will follow. It's just an irrelevant coincidence that I didn't.

Continuing on, we get to `Direct Script Include` which tells me to add two `<script ... />` elements. **Concern:** Where? Is there an example to this? Why don't we put a link to those examples here?

&nbsp;

Following the instructions, as written, is skipping steps and implies some expectations on the audience.
